go.sum

```diff
@@ -1,25 +1,51 @@
 cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
+cloud.google.com/go v0.34.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
@stepanvalyavskiy don't forget to run `go mod tidy` to remove unused ersions, when you finish with dependency issues

Reply:
done